src/Sylius/Bundle/AdminBundle/TwigComponent/Shipment/ShipType.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\TwigComponent\Shipment;
+
+use Sylius\Bundle\AdminBundle\TwigComponent\HookableComponentTrait;
+use Sylius\Component\Core\Model\Shipment;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+use Symfony\UX\LiveComponent\ComponentWithFormTrait;
+use Symfony\UX\LiveComponent\DefaultActionTrait;
+
+#[AsLiveComponent(name: 'SyliusAdmin.Shipment.ShipType', template: '@SyliusAdmin/Shipment/Component/ship.html.twig')]
+final class ShipType
+{
+    use DefaultActionTrait;
+    use HookableComponentTrait;
+    use ComponentWithFormTrait;
+
+    #[LiveProp]
+    public ?Shipment $shipment = null;
+
+    /** @param class-string $formClass */
+    public function __construct(
+        private readonly FormFactoryInterface $formFactory,
+        private readonly string $formClass,
+    ) {
+    }
+
+    protected function instantiateForm(): FormInterface
+    {
+        return $this->formFactory->create($this->formClass, $this->shipment);
+    }
+
+    protected function getDataModelValue(): string
+    {
+        return 'norender|*';
+    }
```

Comment:
What if we remove this? I never had to add it, and I'm just curious why you needed that.

Reply:
Default is `on(change)|*`. 
This form doesn't have constraints, so I think we don't need to send this form every time a user makes a change.